site/src/@ndlib/gatsby-theme-marble/components/Pages/MarbleItem/ItemLayout/TombstoneMetadata/TombstoneField/index.js

```diff
@@ -11,10 +11,18 @@ const TombstoneField = ({ field, sxStyle, searchField }) => {
       {
         field.map((values, i) => {
           return values.value.map((value, j) => {
+            let uriValue = value
+            uriValue = encodeURI(uriValue)
+            uriValue = uriValue.replace(/&/, '%26')
+            uriValue = uriValue.includes(',%20Hesburgh%20Libraries') ? (
```

Comment:
I don't think this is the right place to do this.  

I think this should be done in makeMetadataArray.  

We should add an optional field to the graphql output that is the search facet string.  so that we either put the value of the data or the optional search facet value in the facet link.  
thoughts @danthewolfe ?

Reply:
Yes, I agree. @jonhartzler can you give @jmille79 some guidance on where to make these changes?